bundles/org.openhab.binding.myuplink/src/main/resources/OH-INF/i18n/myuplink.properties

```diff
@@ -0,0 +1,80 @@
+thing-type.config.myuplink.generic-device.deviceId.label = Device Id
+thing-type.config.myuplink.generic-device.deviceId.description = The Id to identify the device.
+thing-type.config.myuplink.generic-device.systemId.label = System Id
+thing-type.config.myuplink.generic-device.systemId.description = The Id of the system the device belongs to.
+
+# channel types
+
+channel-type.myuplink.rwtype-command.label = Generic Command
+channel-type.myuplink.rwtype-mode.label = Generic Command
+channel-type.myuplink.rwtype-mode.state.option.Default = Default
+channel-type.myuplink.rwtype-mode.state.option.Normal = Normal
+channel-type.myuplink.rwtype-mode.state.option.Away = Away
+channel-type.myuplink.rwtype-mode.state.option.Vacation = Vacation
+channel-type.myuplink.rwtype-mode.state.option.Home = Home
+channel-type.myuplink.rwtype-mode.command.option.Default = Default
+channel-type.myuplink.rwtype-mode.command.option.Normal = Normal
+channel-type.myuplink.rwtype-mode.command.option.Away = Away
+channel-type.myuplink.rwtype-mode.command.option.Vacation = Vacation
+channel-type.myuplink.rwtype-mode.command.option.Home = Home
+channel-type.myuplink.rwtype-switch.label = Generic Switch
+channel-type.myuplink.type-electric-current.label = Generic Current
+channel-type.myuplink.type-energy.label = Generic Energy
+channel-type.myuplink.type-flow.label = Generic Flow
+channel-type.myuplink.type-frequency.label = Generic Frequency
+channel-type.myuplink.type-number-double.label = Generic Number (0.1)
+channel-type.myuplink.type-number-integer.label = Generic Number
+channel-type.myuplink.type-on-off.label = Generic OnOff
+channel-type.myuplink.type-on-off.state.option.0 = Off
+channel-type.myuplink.type-on-off.state.option.1 = On
+channel-type.myuplink.type-percent.label = Generic Percentage
+channel-type.myuplink.type-pressure.label = Generic Pressure
+channel-type.myuplink.type-temperature.label = Generic Temperature
+channel-type.myuplink.type-time.label = Generic Time
+
+# thing types config
+
+thing-type.config.myuplink.account.group.customChannels.label = Custom Channels
+thing-type.config.myuplink.account.group.customChannels.description = Custom Channel configuration
+thing-type.config.myuplink.account.group.general.label = General
+thing-type.config.myuplink.account.group.general.description = General settings.
+
+
+# status translations
+
+status.token.validated = "Access token validated"
+status.waiting.for.bridge = "Waiting for bridge to go online"
+status.waiting.for.login = "Waiting for web api login"
+status.no.valid.data = "No valid data received. This is most likely a configuration error"
+status.no.connection = "No connection could be established"
+status.device.not.found = "Device could not be found. Most likely a configuration error"
+status.config.error.no.client.id = "No/Empty client id. Please fix the configuration."
+status.config.error.no.client.secret = "No/Empty client secret. Please fix the configuration."

```

Comment:
Not sure what is going on here, but i see strange line endings that i have not seen before.
![image](https://github.com/user-attachments/assets/ac2145a8-7ab1-46df-8883-187103cfbb98)

Reply:
fixed it